File: app/src/main/java/com/shveed/cookmegood/data/dto/Category.java
package com.shveed.cookmegood.data.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Data;

@Data
@JsonIgnoreProperties({"id", "recipeList"})
public class Category {

    @JsonProperty("name")
    public String name;
}

File: app/src/main/java/com/shveed/cookmegood/data/dto/Category.kt
package com.shveed.cookmegood.data.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Data



@Data
@JsonIgnoreProperties("id", "recipeList")
class Category {
    @JsonProperty("name")
    var name: String? = null
}